.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
```

Comment:
Are we supposed to fill this out or is it just to make it clear that this is a bug report?

If we should fill this out, then I assume that `name` should always have `Bug report` and that `title` will actually describe the issue (e.g. Harbor backup retention is broken). Is that correct?
Is `about` also supposed to just stay as it is, or should we change that to something that describes the bug?

Reply:
So these values are:
name is the name of the template.
about description of the template.
title is what the title of the issue should be prefilled with
labels is the labels that is automatically assigned to the issue
assignees is who is going to be automatically assigned to the issue
Then the stuff under is what the issue body should be prefilled with.

Does that answer your question? (I didn't really understand your concern)

Reply:
I mostly just didn't really understand what this block was used for. I though it would also be visible in the issue body. But after checking some other docs I see that it is used to configure the template and will not be added to the issue body (right?).
So then it is good as it is :slightly_smiling_face: